frontend/src/types.ts

```diff
@@ -1334,15 +1334,15 @@ export interface Group {
 }
 
 export interface Experiment {
-    id: number | null
-    name: string
+    id?: number | null
```

Comment:
I don't think we should relax Experiment typing here. It creates awkward headaches, like testing for params existing before accessing, and is away from reality.

I think you needed this for the `newExperimentData` reducer. Have fixed it by making the reducer type a `Partial<Experiment>`, which I think makes more sense?

Reply:
For draft experiments though, I assume the purpose is to create something and come back to it later - so it could be missing things like name or filters right? Otherwise what is the purpose of enabling a "draft" mode besides not allowing it to start yet? 

I think that was the biggest concern with why passing in Partial<Experiment> wasn't working for the `newExperimentData` reducer

Reply:
Yep, for sure! I made the **return type** of the reducer `Partial<Experiment>`, so it should be okay?

Reply:
..oohh wait, I understand now. It's for the API request as well! 🤦‍♂️ It wouldn't save without all required information being there.

Ugh. To prevent complexity for now, let's just say draft needs all information on saving, and it's basically an intermediary between launching & testing the FF works